.github/workflows/perf.yaml

```diff
@@ -43,12 +43,11 @@ jobs:
         with:
           python-version: ${{matrix.version}}
 
-      - name: Cache Pip Packages
-        uses: actions/cache@1bd1e32a3bdc45362d1e726936510720a7c30a57 # v4.2.0
-        id: cache-pip
-        with:
-          path: ${{ env.PIP_CACHE_DIR }}
-          key: pip-${{ steps.setup_python.outputs.python-version }}-${{ hashFiles('*requirements*.txt') }}
+      - name: Create Python venv
+        run: |
+          python -m venv ${VENV_DIR}
+          source ${VENV_DIR}/bin/activate
+          echo PATH=$PATH >> $GITHUB_ENV
```

Comment:
This does not keep the venv active across job steps from what I can tell.

See that the logs are not using `.turbine-venv`: https://github.com/iree-org/iree-turbine/actions/runs/12714371230/job/35446471647?pr=379#step:5:20

I usually explicitly activate the venv in each step that needs it.

Reply:
I wonder what's going on, `/bin/activate` doesn't do anything special besides updating `PATH` and unsetting `PYTHONHOME`

Reply:
Haha, `echo "$VENV_DIR/bin" >> $GITHUB_PATH` seems to work and it broke torch as it cannot find GPU anymore

Reply:
> That is overwriting PATH, not appending to it?

It's appending, according to docs

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions
```
echo "$HOME/.local/bin" >> "$GITHUB_PATH"
```